pydatastructs/miscellaneous_data_structures/queue.py

```diff
@@ -128,7 +129,7 @@ def __str__(self):
 
 class LinkedListQueue(Queue):
 
-    __slots__ = ['front', 'rear', 'size', '_dtype']
+    __slots__ = ['_dtype']
```

Comment:
`_dtype` can be removed as `LinkedListQueue` should allow elements of different types.

Reply:
duly noted